Sources/SmileID/Classes/Camera/CameraManager.swift

```diff
@@ -166,6 +181,15 @@ class CameraManager: NSObject, ObservableObject, CameraManageable {
             self.session.startRunning()
         }
     }
+
+    internal func capturePhoto() {
+        guard let connection = photoOutput.connection(with: .video), connection.isEnabled, connection.isActive else {
+            return
```

Comment:
Same comment here -- if we're not able to capture a photo, wouldn't this just silently swallow the error? How will the error be communicated back?

Additionally, this would be a good place to add a log